source/manual/deploying-terraform.html.md

```diff
@@ -14,13 +14,13 @@ infrastructure in AWS.
 ## 1. Check what you can deploy
 
 Which changes you can deploy depends on the level of access you have
-to our AWS environments.
+to our AWS environments. Specifically, the level of acces your Amazon Resource Name or ARN has been given.
```

Comment:
Minor typo: `acces`

Reply:
🥇 thanks for the keen eye. Raised in https://github.com/alphagov/govuk-developer-docs/pull/1487.